### PR TITLE
Report player width and height, Add videochange event

### PR DESCRIPTION
**Pull request by nfcampos**

### CODE REVIEW

**Comment by dylanjha** [comment]
@nfcampos this looks really good! I haven't tested it out yet but on first glance I like it. Would have to also update the README with this behavior.

Amazing 🙌

File: README.md [hunk 1]
```diff
@@ -67,11 +70,11 @@ const MuxVideo = muxReactNativeVideo(Video);
...
+1. The `player_is_fullscreen` property is not reported as expected on iOS. The react-native-video callbacks for fullscreen status `onFullscreenPlayerDidPresent` and `onFullscreenPlayerDidDismiss` do not get called when entering fullscreen using the native controls: https://github.com/react-native-video/react-native-video/issues/552
 
 ## Caveats
 
-1. Upscale and downscale % metrics are not calculated because we are unable to get player width and player height measurments from react-native-video.
-1. Even if we could get player width and height, we still wouldn't be able to calculate upscale and downscale % metrics for HLS sources is because of this open issue related to getting the video source width & height: https://github.com/react-native-community/react-native-video/issues/1194.
+1. If we get player width and height, we still won't be able to calculate upscale and downscale % metrics for HLS sources because of this open issue related to getting the video source width & height: https://github.com/react-native-community/react-native-video/issues/1194.
```
Comment (dylanjha) [inline README.md:77]: This bug got fixed, so on more recent versions of react-native-video we should be source width/height now. I'll verify

**Comment by nfcampos** [comment]
I've updated the readme 
If you guys could check the event and properties is as you expect, I tried following the docs, but you'd know best

File: README.md (hunk 1 above)
Comment (nfcampos) [reply README.md:77]: This part didnt get fixed as far as I know https://github.com/react-native-video/react-native-video/issues/1194#issuecomment-596140286

**Comment by dylanjha** [comment]
@nfcampos thank you! Most of the team is on holiday until the new year. We'll check it when we get back. What a great Christmas gift this is 🎁 . Thanks!

**Comment by nfcampos** [comment]
No worries

File: README.md (hunk 1 above)
Comment (dylanjha) [reply README.md:77]: ohh, the part about rendition changes, I think you're right :/ . We should double-check and make sure we're not mis-reporting this number when renditions changes happen. Right now, we do actually track the reported naturalSize (if it is there).

https://github.com/muxinc/mux-stats-sdk-react-native-video/blob/master/src/index.js#L110-L115

**Comment by erikpena** [comment]
Hey @nfcampos,
First of all, thanks for submitting this PR.  We absolutely love it when developers do this :D.  I had taken a look into the width/height reporting (which it looks like you and @dylanjha discussed as well) and I think it's better/safer to omit this from the PR.  Our approach is that we'd rather not have the data than to have data that mis-represents what is actually happening.

I took a deeper dive to see if there was any way would could capture when the player changes renditions so that we can update the width/height but this is simply not exposed within `react-native-video`.

We still want the `video change` work :)

**Comment by nfcampos** [comment]
Alright, I've removed the player width/height changes

**Pull request merged**